modules/exploits/windows/http/nscp_authenticated_rce.rb

```diff
@@ -0,0 +1,228 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Exploit::Remote::HttpServer
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::Powershell
+  prepend Msf::Exploit::Remote::AutoCheck
```

Comment:
```suggestion
  include ::Msf::Exploit::Remote::HttpClient
  include ::Msf::Exploit::Remote::HttpServer
  include ::Msf::Exploit::CmdStager
  include ::Msf::Exploit::Powershell
  prepend ::Msf::Exploit::Remote::AutoCheck
  include ::Rex::Text
```

Includes the `Rex::Text` library and fills out the proper library paths